.coveragerc

```diff
@@ -0,0 +1,6 @@
+[report]
+fail_under = 79
```

Comment:
You can now increase this to 84.

Reply:
@fchollet The current `fail_under` 79 is still good because the test coverages of CNTK are still about 80%.

Reply:
Updated. Sorry for mistake. If this PR is merged, the test coverages of CNTK will be over 84% :)